flocx_market/tests/unit/common/test_prepareservice.py

```diff
@@ -0,0 +1,58 @@
+import flocx_market.conf as conf
+import socket
+import flocx_market.common.service as flocx_market_service
+import pytest
+
+CONF = conf.CONF
+
+
+def test_service_defaults():
+    flocx_market_service.prepare_service()
```

Comment:
Calling this as-is will not always get the result you want; if the person running the test has a /etc/flocx-market/flocx-market.conf configuration file with non-default values, then this test will use those values and fail.

Reply:
I knew this but I wasn't sure how to test defaults otherwise. Do you have any recommendations? Or do you think the test is redundant with tests/conf/test_conf anyways?

Reply:
Can you explicitly point it at a non-default config file (like `/dev/null`)?

Reply:
Tested this out a bit, and adding an import and changing the test a bit works:

```
import unittest.mock as mock

@mock.patch('oslo_config.cfg.find_config_files')
def test_service_defaults(find_config_files):
      .
      .
```

I'd recommend doing that for all of these tests.

Reply:
Just for clarity, oslo_config.cfg.find_config_files is what searches for config files in certain default locations; mocking that method guarantees it will always return nothing, forcing the tests to rely on the defaults.